File: Latihan_b/Python/Disk.py
class Disk:
    
    # atribut
    __type = ""
    __capacity = ""
    __price = 0

    # konstruktor

    def __init__(self, type, capacity, price):
        self.__type = type
        self.__capacity = capacity
        self.__price = price
    

    # setter
    def setType(self, type):
        self.__type = type
    
        
    def setCapacity(self, capacity):
        self.__capacity = capacity
    

    def setPrice(self, price):
        self.__price = price
    

    # getter
    def getType(self):
        return self.__type
    
        
    def getCapacity(self):
        return self.__capacity
    

    def getPrice(self):
        return self.__price
    

File: Latihan_b/Java/Disk.java
class Disk {

    // atribut
    private String type;
    private String capacity;
    private int price;

    // konstruktor
    public Disk(){}

    public Disk(String type, String capacity, int price)
    {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    // setter
    public void setType(String type)
    {
        this.type = type;
    }
        
    public void setCapacity(String capacity)
    {
        this.capacity = capacity;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // getter
    public String getType()
    {
        return this.type;
    }
        
    public String getCapacity()
    {
        return this.capacity;
    }

    public int getPrice()
    {
        return this.price;
    }
}